tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h

```diff
@@ -314,14 +322,15 @@ __device__ T ReduceBlockAlongCols(ReduceOp reduce_op, const T& value,
 // size (as long as it spans at least the maximum value in indices). This also
 // applies to the weights vector.
 template <typename Treducevec, typename Tvec, typename Tindex,
-          typename Tsegmentids, typename ReduceOp, typename Tinit>
+          typename Tindices, typename Tsegmentids, typename ReduceOp,
```

Comment:
It seems weird to have a `Tindex` and `Tindices`.  Can we be more specific, or rename `Tindex` to `Toffsets`  or similar?

Reply:
Done, renamed `Tindex` to `Toffsets`.